packages/website/src/bin/page-generator/get-docs-info.test.js

```diff
@@ -64,3 +64,34 @@ describe('Missing docs directory', () => {
     expect(docsInfo).toBeNull();
   });
 });
+
+describe('docs-with-readmes', () => {
+  let cwd;
+  let docsInfo;
+
+  beforeAll(async () => {
+    cwd = await copyFixtureIntoTempDir(__dirname, 'docs-with-readme');
+    docsInfo = getDocsInfo(path.join(cwd, 'docs'));
+  });
+
+  it('should replace the base index page with the readme page', () => {
+    expect(docsInfo[2]).toEqual({
+      id: 'doc-3',
+      children: [
+        {
+          id: 'doc-3-2',
+          children: [
+            {
+              id: 'README',
+              path: path.join(cwd, 'docs', 'doc-3', 'doc-3-2', 'README.md'),
+            },
+          ],
+        },
+        {
+          id: 'readme',
```

Comment:
is using this id a safe way to determine what type of page this is? What happens if i have a directory called `readme`?

Reply:
For now, yes. This is fine.

A readme dir will have `children` which will make it distinguishable.

Maybe followup should be having both a folder and a file with a name - what does this do? solve for that.